src/python/strelka/scanners/scan_x509.py

```diff
@@ -15,7 +15,7 @@ class ScanX509(strelka.Scanner):
             scanned. Must be either 'der' or 'pem'.
             Defaults to empty string.
     """
-    def scan(self, data, file, options, expire_at):
+    def scan(self, data, file, options, expire_at, _):
```

Comment:
Not a big deal but I figured I'd ask:

With this change we'll see some disruption if we merge from `target/strelka`, e.g. any new scanners will require modification. We're pretty far behind anyways and I know we'd have a bunch of conflicts, but is there any way to avoid adding this param? Maybe adding to `options`?

Reply:
Much better, thanks!